build.gradle.kts

```diff
@@ -42,6 +44,12 @@ application {
 	}
 }
 
+java {
+	sourceSets.main.configure {
+		java.srcDir(layout.buildDirectory.dir("generated/libgit2"))
```

Comment:
When I point it to the task, instead, I get:
```
* What went wrong:
Could not determine the dependencies of task ':distTar'.
> Could not resolve all dependencies for configuration ':runtimeClasspath'.
   > Cannot invoke "Object.getClass()" because "sources" is null
```